lib/ve_logger_filter.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+module.exports = (hyper, req, next, options, specInfo) => {
+
+    options.probability = options.probability || 1;
+
+    if (req.headers && req.headers['user-agent'] &&
+            req.headers['user-agent'].includes('VisualEditor') &&
```

Comment:
Better check for `Api-User-Agent`, browser will override the user-agent

Reply:
VE sets both, but yeah, can check that one too